src/sentry/spans/buffer.py

```diff
@@ -255,8 +255,8 @@ def process_spans(self, spans: Sequence[Span], now: int):
         metrics.timing("spans.buffer.process_spans.num_spans", len(spans))
         metrics.timing("spans.buffer.process_spans.num_is_root_spans", is_root_span_count)
         metrics.timing("spans.buffer.process_spans.num_has_root_spans", has_root_span_count)
-        metrics.timing("span.buffer.hole_size.min", min_redirect_depth)
-        metrics.timing("span.buffer.hole_size.max", max_redirect_depth)
+        metrics.timing("spans.buffer.redirect_depth.min", min_redirect_depth)
+        metrics.timing("spans.buffer.redirect_depth.max", max_redirect_depth)
```

Comment:
Since this is a timing, it creates rather strange `redirect_depth.min.max` and `redirect_depth.max.min` metrics. 

@untitaker, what if we instead log both of these values under `spans.buffer.redirect_depth` and then just query the resulting avg, min, and max of that timer?

Reply:
you can also change the type to gauge, then it is equivalent to the min/max metrics we have in prod

Reply:
I think gauges always keep the last reported value, so we'd lose the true max and min.